docs/2.0/integration/behave.adoc

```diff
@@ -0,0 +1,30 @@
+= Behave
+Allure integrate with behave as an external formatter.
+
+== Instalation
+[source, bash]
+----
+$ pip install allure-behave
+----
+
+== Usage
+You can specify the formatter directly in the command line:
+
+[source, bash]
+----
+$ behave -f allure_behave.formatter:AllureFormatter -o %allure_result_folder% ./features
+----
+
+== Features
+
+=== Severity
+Tags that matched to severity names, like critical, trivial, etc, will be interpreted as feature or scenario severity.
+Scenario inherite a feature severity, the same way, scenario severity is overdride a feature severity. When situated
```

Comment:
Scenario inherits feature severity if not provided, or overrides it in the other case. If there is more than one severity definition tag, only the last one is used.